<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ReviewSearchDialog</class>
 <widget class="QDialog" name="ReviewSearchDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>700</width>
    <height>240</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ReviewSearchDialog</string>
  </property>
  <widget class="QLabel" name="lbSampleId">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>80</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>Sample ID</string>
   </property>
  </widget>
  <widget class="SampleIdLineEdit" name="leSampleId">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>20</y>
     <width>200</width>
     <height>35</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="lbGender">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>20</y>
     <width>80</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>Gender</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="rbMale">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>20</y>
     <width>90</width>
     <height>35</height>
    </rect>
   </property>
   <property name="text">
    <string>Male</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="rbFemale">
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>20</y>
     <width>90</width>
     <height>35</height>
    </rect>
   </property>
   <property name="text">
    <string>Female</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbPatientId">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>70</y>
     <width>80</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>Patient ID</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lePatientId">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>70</y>
     <width>200</width>
     <height>35</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="lbName">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>70</y>
     <width>80</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>Name</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="leName">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>70</y>
     <width>200</width>
     <height>35</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="lbSender">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>120</y>
     <width>80</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>Sender</string>
   </property>
  </widget>
  <widget class="QComboBox" name="cbSender">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>120</y>
     <width>200</width>
     <height>35</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="lbSendDate">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>120</y>
     <width>80</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>Send Date</string>
   </property>
  </widget>
  <widget class="DateLineEdit" name="leSendDate">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>120</y>
     <width>200</width>
     <height>35</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pbAll">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>180</y>
     <width>120</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>All</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pbSearch">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>180</y>
     <width>120</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>Search</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pbCancel">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>180</y>
     <width>120</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>LineEdit</class>
   <extends>QLineEdit</extends>
   <header>lineedit.h</header>
  </customwidget>
  <customwidget>
   <class>DateLineEdit</class>
   <extends>LineEdit</extends>
   <header>datelineedit.h</header>
  </customwidget>
  <customwidget>
   <class>SampleIdLineEdit</class>
   <extends>LineEdit</extends>
   <header>sampleidlineedit.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
